Cargo.toml

```diff
@@ -19,6 +19,7 @@ tracing-error = { version = "0.2.0", optional = true }
 tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"], optional = true }
 directories = { version = "5.0.1", optional = true }
+tui-widgets = "0.3.0"
```

Comment:
Maybe we could depend on `tui-scrollview` directly instead of pulling in more unnecessary widgets.
Also, please make this dependency optional and add it to the `cli` feature below.